src/plugins/imagery/components/ImageryView.vue

```diff
@@ -774,10 +905,87 @@ export default {
             let index = this.focusedImageIndex;
 
             if (index === this.imageHistory.length - 1) {
-                this.setFocusedImage(this.imageHistory.length - 2, THUMBNAIL_CLICKED);
+                this.thumbnailClicked(this.imageHistory.length - 2);
             } else {
-                this.setFocusedImage(--index, THUMBNAIL_CLICKED);
+                this.thumbnailClicked(--index);
+            }
+        },
+        resetImage(overrideLock) {
+            if (this.panZoomLocked && !overrideLock) {
```

Comment:
We shouldn't need any references to zoomFactor or panZoomLocked here after the image controls are moved into the sub component